lib/ultrajson.h

```diff
@@ -259,6 +259,7 @@ typedef struct __JSONObjectEncoder
   /*
   Configuration for spaces of indent */
   int indent;
+  const char* indent_chars;
```

Comment:
The comment should be expanded a bit on what the exact meaning of these two values is.

I'd also like to propose renaming `indent` to `indentLength` to make the meaning clearer but would like to hear what others think about that.

Reply:
I like `indentLength`. Only reason I didn't change it in the first place was to minimize the diff. But I'll add that.